ArcanicVosforator/src/main/java/org/example/arcanicvosforator/ArcanicVosforatorInterface.java

```diff
@@ -941,27 +654,67 @@ private void selectArcaneEvent() {
         }
     }
 
+    private void addToVosforCount(int value, int quantity){
+        getData().addToTotalVosforCount(getData().getVosforRates().get(value), quantity);
+        getData().addToGrandTotalVosforCount();
+    }
+
     private void clearDataEvent() {
         getData().clearTotalVosforCount();
         getData().clearGrandTotalVosforCount();
         arcaneQuantity.clear();
         listOfArcanes.clear();
         label.setText("");
         arcaneSetList.setItems(null);
-        eidolonArcaneSetFilter.setSelected(false);
-        cetusArcaneSetFilter.setSelected(false);
-        arbitrationArcaneSetFilter.setSelected(false);
-        steelPathArcaneSetFilter.setSelected(false);
-        zarimanArcaneSetFilter.setSelected(false);
-        quillsArcaneSetFilter.setSelected(false);
-        necraliskArcaneSetFilter.setSelected(false);
-        duviriArcaneSetFilter.setSelected(false);
-        caviaArcaneSetFilter.setSelected(false);
-        luaArcaneSetFilter.setSelected(false);
-        fortunaArcaneSetFilter.setSelected(false);
-        mirrorDefenseArcaneSetFilter.setSelected(false);
-        plagueStarArcaneSetFilter.setSelected(false);
-        voxSolarisArcaneSetFilter.setSelected(false);
+        clearSelected();
```

Comment:
you can take or leeave this one. just really quickly popped this one in here too.